src/rules/no-restricted-html-elements.ts

```diff
@@ -0,0 +1,62 @@
+import { createRule } from "../utils"
+
+export default createRule("no-restricted-html-elements", {
+  meta: {
+    docs: {
+      description: "disallow specific HTML elements",
+      category: "Extension Rules",
```

Comment:
Hmm. I think the better category for this rule is `Stylistic Issues` among the categories we currently have, but we may need to create a new category 🤔

Reply:
Ah, I misunderstood the meaning of `Extension Rules`.
Just now I changed the category.
Should we refactor categories in the next major version?

Reply:
It looks like we need to run `pnpm run update`.

Reply:
> Should we refactor categories in the next major version?

We can always make changes if we have ideas for categorization that make it easier for users to understand 😉

Reply:
I thought about it a bit, but I think `Stylistic Issues` is fine for now.